rules/xcodeproj.bzl

```diff
@@ -1,7 +1,7 @@
 load("@build_bazel_rules_apple//apple:providers.bzl", "AppleBundleInfo")
 load("@bazel_skylib//lib:paths.bzl", "paths")
 
-def _get_transitive(deps, provider, field):
+def _get_attr_values_for_name(deps, provider, field):
```

Comment:
WDYT of this method rename? I thought this name was easier to understand, but open to hearing other options :)